### ISSUE TITLE
disappering images

### ISSUE THREAD

**Issue by Julime** [RocketChat/Rocket.Chat#2544]
Rocket.Chat version: 0.22.0
Since the last update images randomly disapper and appear when looking at them.
A reload sometimes fixes this for a few seconds. You can still view them in the files menu (right sidebar).
This issue does only take affect on uploaded images, not link previews.
![capturfiles_1](https://cloud.githubusercontent.com/assets/7944656/13873133/a1a14414-ecef-11e5-981e-1764ab5f0915.png)

**Comment by Julime** [comment]
![capturfiles_2](https://cloud.githubusercontent.com/assets/7944656/13873161/e82d8ece-ecef-11e5-963e-2128b39da75b.png)

**Comment by engelgabriel** [comment]
Are you using GridFS?

**Comment by Julime** [comment]
im using the docker mongodb image:
```
mongo:
  image: mongo
  volumes:
    - ./data/runtime/db:/data/db
    - ./data/dump:/dump
  command: mongod --smallfiles --oplogSize 128
  restart: always
```

**Comment by engelgabriel** [comment]
Do you see any errors on the console or network tabs of your browser's dev tools?

**Comment by Julime** [comment]
![capturfiles_3](https://cloud.githubusercontent.com/assets/7944656/13880032/b7598548-ed1a-11e5-9ee2-3230cc382bcb.png)

**Comment by graywolf336** [comment]
Probably a duplicate of #2545 as well.

**Comment by manuelbachl** [comment]
I'm pretty sure it's the same bug as #2243

**Comment by sampaiodiego** [comment]
This is probably related to #2545
@Julime are you using GridFS as file storage? check at `Administration > File Upload > Storage Type`.

**Comment by Julime** [comment]
thx @sampaiodiego . I'm using GridFS

**Comment by sampaiodiego** [comment]
ok, we'll merge #2568 and make a new release today.
I hope it fix your problem :smile:

**Comment by engelgabriel** [comment]
This should be fixed on version 0.23.0
Please reopen otherwise.
